RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc

```diff
@@ -0,0 +1,464 @@
+/**

```

Comment:
General comment: it would be nice to start the code for this new algorithm on the right foot by having a consistent indentation style within the file (currently 2 spaces in some places, tabs in other places, etc.)